ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/CV_PATHFINDER_TESTS.yml

```diff
@@ -76,7 +76,7 @@ wan_transit:
   defaults:
     loopback_ipv4_pool: 192.168.43.0/24
     filter:
-      always_include_vrfs_in_tenants: [TenantA]
+      always_include_vrfs_in_tenants: [TenantA, TenantB]
```

Comment:
We may have to consider to always include all VRFs on pathfinder and transit. Or somehow give easy control of which VRFs are part of the WAN. (Maybe a key under the VRFs to exclude it.

Reply:
not on transit I think and even for pathfinder it may be relevant only if one connected node has it - not 100% sure :)

Reply:
I don't think we need the vrfs on pathfinders at all (only default). But for transit I think we _will_ need all the VRFs that are covered in the region, and easiest would be to just take all of them.

Reply:
We will keep the current logic for now. We can revisit VRF layout once we get some feedback.